src/components/RainWindInfo/WindDescription.tsx

```diff
@@ -13,15 +14,15 @@ export type WindSpeedDescription = {
 };
 
 const WindSpeedValues: WindSpeedDescription[] = [
-  { min: 0, max: 0.499, label: "no wind" },
-  { min: 0.5, max: 5.399, label: "light wind" },
-  { min: 5.4, max: 7.999, label: "medium wind" },
-  { min: 8, max: 13.799, label: "strong wind" },
-  { min: 13.8, max: 16.999, label: "very strong wind" },
-  { min: 17, max: 20.799, label: "storm" },
-  { min: 20.8, max: 24.499, label: "strong storm" },
-  { min: 24.5, max: 29.999, label: "violent storm" },
-  { min: 30, max: 1000, label: "hurricane" }
+  { min: 0, max: 0.499, label: "windDescr_1" },
+  { min: 0.5, max: 5.399, label: "windDescr_2" },
+  { min: 5.4, max: 7.999, label: "windDescr_3" },
+  { min: 8, max: 13.799, label: "windDescr_4" },
+  { min: 13.8, max: 16.999, label: "windDescr_5" },
+  { min: 17, max: 20.799, label: "windDescr_6" },
+  { min: 20.8, max: 24.499, label: "windDescr_7" },
+  { min: 24.5, max: 29.999, label: "windDescr_8" },
+  { min: 30, max: 1000, label: "windDescr_9" }
 ];
```

Comment:
Just a convention detail, generally it is a good idea to move away from naming like "label", "title", "name" when you are passing i18n key.

I suggest changing `label` to `i18n_key_label`/`i18nKeyLabel` or something similar.

The reason behind that is that both of those strings are just strings and before seeing the output in the app (or checking declaration)  you don't know if this "label" is something that you have to run through translation utility.